systest/tests/checkpoint_test.go

```diff
@@ -169,15 +169,15 @@ func TestCheckpoint(t *testing.T) {
 	}
 
 	tctx.Log.Infow("waiting for all miners to be smeshing", "last epoch", checkpointEpoch+2)
-	ensureSmeshing(t, tctx, cl, checkpointEpoch+2)
+	ensureSmeshing(t, tctx, cl, checkpointEpoch+4)
```

Comment:
I think we can just change the value of `checkpointEpoch` to `8` instead of `4`. Also the log line above should probably say:

```go
tctx.Log.Infow("waiting for all miners to be smeshing", "checkpoint epoch", checkpointEpoch)
```